aiac/test/pdp/service/policy/opa/test_main.py

```
@@ -19,6 +19,28 @@
+
+Note on the delete-by-id endpoint: its route param ``{agent_id}`` is a single
+path segment, and a valid namespaced id (``<ns>/<name>`` or a SPIFFE URI) carries
+slashes. The library client percent-encodes them and the ASGI server decodes the
+segment back, but the ``TestClient``/httpx transport collapses ``%2F`` -> ``/``
+before the request is sent, so a namespaced id cannot reach the param through
+``TestClient``. Those cases therefore call the route handler function directly
+(the FastAPI decorators leave the functions callable), which still exercises the
+full write + error-mapping path through the mocked ``_api``.
+"""
+
+import json
```

Ruff (0.16.2)

[error] Module level import not at top of file

(E402)